fe/src/layouts/default.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div>
+    <header class="shadow-sm bg-white">
+      <nav class="container m-auto py-2 px-5 flex justify-between items-center">
+        <NuxtLink to="/" class="justify-center"><Logo /></NuxtLink>
+        <div class="flex gap-4">
+          <BaseSearch />
+          <WalletButton :connected="connected" :address="accountAddress" />
+        </div>
+      </nav>
```

Comment:
Lets make a navbar component for this one.
```suggestion
      <BaseNavbar />
```